fury/actor.py

```diff
@@ -247,6 +258,138 @@ def sphere(
     )
 
 
+def ellipsoid(
+    centers,
+    *,
+    axes=None,
+    lengths=(4, 3, 2),
+    colors=(1, 0, 0),
+    opacity=None,
+    phi=16,
+    theta=16,
+    material="phong",
+    enable_picking=True,
+    smooth=True,
+):
+    """
+    Create ellipsoid actor(s) with specified orientation and scaling.
+
+    Parameters
+    ----------
+    centers : ndarray (N, 3)
+        Centers of the ellipsoids
+    axes : ndarray, shape (N, 3, 3) or (3, 3), optional
+        Rotation matrices for ellipsoids. Each 3×3 matrix defines a local
+        coordinate frame, where the columns represent the ellipsoid’s x, y, and z
+        axes in world space. This determines the ellipsoid's orientation.
+        Must be orthonormal and right-handed.
+    lengths : ndarray (N, 3) or (3,) or tuple (3,), optional
+        Scaling factors along each axis
+    colors : array-like or tuple, optional
+        RGB/RGBA colors for each ellipsoid
+    opacity : float, optional
+        Overall transparency (0-1)
+    phi : int, optional
+        Latitude divisions
+    theta : int, optional
+        Longitude divisions
+    material : str, optional
+        Surface material type
+    enable_picking : bool, optional
+        Allow mouse selection
+    smooth : bool, optional
+        Smooth shading
+
+    Returns
+    -------
+    actor : fury.actor.Actor
+        Ellipsoid actor with transformations applied
+
+    Examples
+    --------
+    >>> from fury import window, actor
```

Comment:
Example uses the show method incorrectly.

We use show with list of actors now.

Correct version
```
import numpy as np

from fury import actor, window

centers = np.array([[0, 0, 0]])
lengths = np.array([[2, 1, 1]])
colors = np.array([[1, 0, 0]])
ellipsoid = actor.ellipsoid(centers=centers, lengths=lengths, colors=colors)
window.show([ellipsoid])
```